.vscode/launch.json

```diff
@@ -17,7 +17,7 @@
                 "--config",
                 "dfastmi.cfg",
                 "--rivers",
-                "Dutch_rivers_v2.ini"
+                "..\\tests\\c01 - GendtseWaardNevengeul\\rivers_Q4000_v2.ini"
```

Comment:
Should this change stay?

Reply:
Well yes. because I can do a run in the GUI now. When I set this in the master (so before my changes) I can do a successful run too. This will help in the development of the GUI